AdornUI-Stylesheets/utility.css

```diff
@@ -0,0 +1,8 @@
+.flex-vt{
+    display: flex;
+    flex-direction: column;
+}
+.flex-hz{
+    display: flex;
+    flex-direction: row;
```

Comment:
I will suggest making a class for flex properties and using it where ever you need it, it will reduce code repetition.